### ISSUE TITLE
Remove 'seconds' from timestamps

### ISSUE THREAD

**Issue by zoeyli-46** [bcgov/cas-registration#2640]
#### Describe the task
The date/timestamp is long which affects the readability. 
- [ ] Remove the 'seconds' from the timestamps so that it will be more readable. i.e., July 4, 2025, 4:25 p.m. PDT
<img width="738" alt="Image" src="https://github.com/user-attachments/assets/4e7831d1-eb85-4857-9bde-9b7c8917b1a8" />

#### Acceptance Criteria

- [ ] first
- [ ] second
- [ ] third

#### Definition of Done

- [ ] first
- [ ] second
- [ ] third

#### Additional context

- Add any other context about the task here.
- Or here

**Comment by zoeyli-46** [comment]
@patriciarussellCAS I'd advocate for this to be fixed for MVP if it would be a quick fix from developers

**Comment by patriciarussellCAS** [comment]
Thanks @zoeyli-46 that makes sense. 
Is this ticket only for the transfer table? Or more broad for all the places we have a date/time stamp? 
If this is only for the transfer table should we document the decision in the design guidelines or somewhere else to ensure the rule is applied consistently across the app?

**Comment by zoeyli-46** [comment]
This ticket is for everywhere there is a "timestamp". Yes, indeed we have it in the [design guidelines](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=29420-6422&t=EfQYkDfFzF6JYjzk-4)!

**Comment by fviduya** [comment]
I have tested this in the DEV environment, [here is the test document for reference](https://bcgov-my.sharepoint.com/:w:/g/personal/francesca_viduya_gov_bc_ca/ETBYpSCP0tdKqmLQO08dNRgBQ5nrBfhhNXXGgUZ7wCa46A?e=HenMEh).

In the test document, I was able to note only two scenarios where we have a timestamp: When signing off a report and when transferring an operation/facility. Is there any more I missed?

I was able to verify that 'seconds' was removed from the timestamp for both scenarios in the document, however, based on the design guideline the date should be written as 'Month Date Year' which is not how it currently looks like when signing off a report (It uses YYYY-MM-DD). I also wanted to clarify regarding the time if it should be in 24hr format, or 12hr format, as the two scenarios I tested were showing different formats. Transferring to Pending PO for additional information and verification of test.

cc: @vesselak / @patriciarussellCAS

**Comment by patriciarussellCAS** [comment]
Thanks Chesca. @NicoleGovvy do we have any guidelines on what is standard for date/time formats?

**Comment by NicoleGovvy** [comment]
@patriciarussellCAS I found a bit of info for [BC API standards](https://developer.gov.bc.ca/docs/default/component/bc-developer-guide/bc-government-api-guidelines/#encoding-and-metadata), but it seems like the format can be modified if needed: 

![Image](https://github.com/user-attachments/assets/6ef30c4e-786b-49a3-b240-71958ee04caa)

**Comment by patriciarussellCAS** [comment]
@fviduya I think we should make them all consistent and use the app design guidelines, so month, day, year. Are you able to do this?

**Comment by fviduya** [comment]
Yes, I can do that. I'll follow the app design guidelines and use the 12hr format as well!
